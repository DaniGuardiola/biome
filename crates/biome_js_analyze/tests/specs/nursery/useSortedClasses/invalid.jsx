<div class="px-4 p-4" />;
