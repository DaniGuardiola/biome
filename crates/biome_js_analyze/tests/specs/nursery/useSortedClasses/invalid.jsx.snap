---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: invalid.jsx
---
# Input
```jsx
<div class="px-4 p-4" />;

```

# Diagnostics
```
invalid.jsx:1:12 lint/nursery/useSortedClasses  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ! These CSS classes should be sorted.
  
  > 1 │ <div class="px-4 p-4" />;
      │            ^^^^^^^^^^
    2 │ 
  
  i Safe fix: Sort the classes.
  
    1   │ - <div·class="px-4·p-4"·/>;
      1 │ + <div·class="p-4·px-4"·/>;
    2 2 │   
  

```


